staging/cli.c

```diff
@@ -525,6 +525,9 @@ int cli(void)
    
    // Create a socket
    s = socket(AF_INET, SOCK_STREAM, 0);
+   if (s==-1) {
+	   perror("socket");
```

Comment:
I think we should return here instead of keep on working on the invalid socket `s` further down.